From f207c1147c1ca6ea1ad1dc3d3d19053ab8352df6 Mon Sep 17 00:00:00 2001
From: Terry Lv <r65388@freescale.com>
Date: Thu, 19 Jul 2012 16:40:12 +0800
Subject: [PATCH 334/505] ENGR00217628: Correct test doc path and syntax error
 of caam

Correct test doc path and syntax error of caam.

Signed-off-by: Terry Lv <r65388@freescale.com>
---
 test/caam/doc/FSL-UT-CAAM-001.txt | 4 ++--
 test/caam/doc/FSL-UT-CAAM-002.txt | 4 ++--
 test/caam/doc/FSL-UT-CAAM-003.txt | 4 ++--
 test/doc/ut_um.toc                | 6 +++---
 4 files changed, 9 insertions(+), 9 deletions(-)

diff --git a/test/caam/doc/FSL-UT-CAAM-001.txt b/test/caam/doc/FSL-UT-CAAM-001.txt
index 2610953..35d0997 100644
--- a/test/caam/doc/FSL-UT-CAAM-001.txt
+++ b/test/caam/doc/FSL-UT-CAAM-001.txt
@@ -1,5 +1,5 @@
-FSL-UT-CAAM-001-DES
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+FSL-UT-CAAM-001
+~~~~~~~~~~~~~~~
 
 [cols=">s,6a",frame="topbot",options="header"]
 |====================================================================
diff --git a/test/caam/doc/FSL-UT-CAAM-002.txt b/test/caam/doc/FSL-UT-CAAM-002.txt
index 071c045..6ca9f4e 100644
--- a/test/caam/doc/FSL-UT-CAAM-002.txt
+++ b/test/caam/doc/FSL-UT-CAAM-002.txt
@@ -1,5 +1,5 @@
-FSL-UT-CAAM-001-DES
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+FSL-UT-CAAM-002
+~~~~~~~~~~~~~~~
 
 [cols=">s,6a",frame="topbot",options="header"]
 |====================================================================
diff --git a/test/caam/doc/FSL-UT-CAAM-003.txt b/test/caam/doc/FSL-UT-CAAM-003.txt
index 807fa96..0d75c73 100644
--- a/test/caam/doc/FSL-UT-CAAM-003.txt
+++ b/test/caam/doc/FSL-UT-CAAM-003.txt
@@ -1,5 +1,5 @@
-FSL-UT-CAAM-001-DES
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+FSL-UT-CAAM-003
+~~~~~~~~~~~~~~~
 
 [cols=">s,6a",frame="topbot",options="header"]
 |====================================================================
diff --git a/test/doc/ut_um.toc b/test/doc/ut_um.toc
index ca52974..a3beead 100644
--- a/test/doc/ut_um.toc
+++ b/test/doc/ut_um.toc
@@ -11,9 +11,9 @@ include::../mxc_asrc_test/doc/FSL-UT-ASRC-002.txt[]
 
 CAAM
 ----
-include::../can/doc/FSL-UT-CAAM-001.txt[]
-include::../can/doc/FSL-UT-CAAM-002.txt[]
-include::../can/doc/FSL-UT-CAAM-003.txt[]
+include::../caam/doc/FSL-UT-CAAM-001.txt[]
+include::../caam/doc/FSL-UT-CAAM-002.txt[]
+include::../caam/doc/FSL-UT-CAAM-003.txt[]
 
 CAN
 ---
-- 
1.8.0

